pkg/oonf_api/0001-add-RIOT-support.patch

```diff
@@ -91,10 +91,10 @@ index 77c519b..e09c0c9 100644
 +#endif
 +
  #include "common/autobuf.h"
-
+ 
```

Comment:
Some versions of git (my local e.g.) register patch files as binary files and thus ignore their content for patching, diff and whitespace checks.